eth/client.go

```diff
@@ -31,7 +31,7 @@ import (
 )
 
 var (
-	RoundsPerTokenSharesClaim = big.NewInt(50)
+	RoundsPerEarningsClaim = big.NewInt(50)
```

Comment:
This value will have to match the value set for the `maxEarningsClaimsRounds` parameter for the BondingManager contract. The default during deployment is currently 20 (https://github.com/livepeer/protocol/blob/master/migrations/migrations.config.js)